exe/solr_wrapper

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env ruby
 
+if ENV['RAILS_ENV'] && ENV['RAILS_ENV'] == 'production'
```

Comment:
I don't think it's an appropriate check for this gem to make. While I think it's a bad idea for someone to want to do that, this gem has no place enforcing that.